docs/quickstart/customize.md

```diff
@@ -18,3 +20,24 @@ Updating static files...
 Go to the browser [*https://localhost/*](https://localhost) or refresh the page. And voila! The logo has been changed!
 
 **WARNING**: If you do not see it changing, check in an incognito window, the browser might have cached the logo.
+
+## CSS + Templates
```

Comment:
Moderate: I would remove the CSS part just not to give false hopes. We have a series of issues with webpack, watchpack and invenio-assets (see https://github.com/inveniosoftware/invenio-assets/commit/094c1ef4d0c67b46ab890d8303d51bab1782ba55 and https://github.com/inveniosoftware/invenio-app-rdm/issues/5 for more info)

Reply:
Makes sense. Let's add it back when those issues are solved.